modules/hostap/src/supp_api.c

```diff
@@ -1008,3 +1008,269 @@ int supplicant_ap_sta_disconnect(const struct device *dev,
 	return ret;
 }
 #endif /* CONFIG_AP */
+
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_DPP
```

Comment:
You can remove this so that the DPP code gets compile tested even if it is not used.